js/app.js

```diff
@@ -35,182 +35,218 @@
 let score = 0;
 let playerName = prompt('What is your name?');
 console.log(playerName);
-alert('Hello, ' + playerName + '. Please answer the following questions simply with a yes or a no.');
+alert(
+  'Hello, ' +
+    playerName +
+    '. Please answer the following questions simply with a yes or a no.'
+);
 
 // Question 1 -
-let a1 = prompt('Are my pronouns he/him?').toLowerCase();
-if (a1 === 'no') {
-  ////console.log('correct');
-  alert('Correct!');
-  score++;
-  console.log(score);
-} else if (a1 === 'yes') {
-  ////console.log('incorrect');
-  alert('Incorrect!');
-} else {
-  //console.log('Not an answer.');
-  alert('That is not an answer.');
+function q1() {
+  let a1 = prompt('Are my pronouns he/him?').toLowerCase();
+  if (a1 === 'no') {
+    ////console.log('correct');
+    alert('Correct!');
+    score++;
+    console.log(score);
+  } else if (a1 === 'yes') {
+    ////console.log('incorrect');
+    alert('Incorrect!');
+  } else {
+    //console.log('Not an answer.');
+    alert('That is not an answer.');
+  }
 }
-
+q1();
```

Comment:
Try to have all your function calls and the end of your document. That way other people looking at your code can see what was called and in what order. It also helps a lot with bug fixing!